src/App.js

```diff
@@ -39,13 +54,24 @@ class App extends Component {
               </Navbar.Brand>
               <Navbar.Toggle />
             </Navbar.Header>
+            <ToggleButtonGroup
```

Comment:
Maybe `ToggleButtonGroup` isn't the best to use here, since it is more geared towards actually having a group. We can leave it for now until we find a better single toggle/switch solution, but I'd leave a `TODO` comment for that